### PR TITLE
Tools: install-prereqs-ubuntu: update on Ubuntu 16.x

### PR DESCRIPTION
**Pull request by khancyr**
Add libtool-bin on Ubuntu 16.04 otherwise jsbsim failed to build
Without : 
````
which libtool
libtool not found
````
With : 
````
which libtool               
/usr/bin/libtool
````

### CODE REVIEW

**Comment by lucasdemarchi** [comment]
I'm not sure this is a difference from ubuntu 16.04 to previous versions... libtool was not explicitly installed before, probably because it's not our dependency, but rather jsbsim's.

Does it really need libtool-bin and not just libtool package?  Searching for ubuntu packages:
http://packages.ubuntu.com/search?keywords=libtool

It seems libtool is available in most of the versions.

**Comment by khancyr** [comment]
I catch this error from here : http://ardupilot.org/dev/docs/setting-up-sitl-on-linux.html#jsbsim-plane-only

And in install-prereqs-ubuntu.sh, we already install libtool in px4_pkgs, libtool is need for Plane Sitl, so we need it .
On ubuntu 14.04, I didn't get error but, starting with Wily, I start to need libtool-bin. And in 16.04 we need too.

Look in package, we can see that they remove libtool binary from libtool
http://packages.ubuntu.com/xenial/all/libtool/filelist
http://packages.ubuntu.com/trusty/amd64/libtool/filelist

**Comment by OXINARF** [comment]
@khancyr If libtool-bin was already needed in 15.10 we should install the package there too. Let's use the Ubuntu version instead of name and put an if with a _greater than_ around the existing one.

**Comment by lucasdemarchi** [comment]
@OXINARF agreed. Also I don't see libtool belonging to the PX4 camp, but rather SITL packages.

**Comment by lucasdemarchi** [comment]
@khancyr any news on this?

**Comment by khancyr** [comment]
will rework that this afternoon !

**Comment by khancyr** [comment]
Hi,

Got some other questions : 
# arduino-core ? seems not need
# curl needed for ?
# droneapi needed for ?
# catkin_pkg needed for ? It is ROS related but actually we don't use ROS.
# g++ ccache should be in base_pkgs

File: Tools/scripts/install-prereqs-ubuntu.sh [hunk 1]
```diff
@@ -2,21 +2,37 @@
...
+SITL_PKGS="python-pip python-setuptools python-matplotlib python-serial python-scipy python-opencv python-numpy python-pyparsing realpath"
 ASSUME_YES=false
 
-read -r UBUNTU_CODENAME <<<$(lsb_release -c -s)
+UBUNTU_YEAR="15" # Ubuntu Year were changes append
+UBUNTU_MONTH="10" # Ubuntu Month were changes append
 
-if [ "$UBUNTU_CODENAME" = "xenial" ]; then
-    SITL_PKGS+=" python-wxgtk3.0"
+version=$(lsb_release -r | awk '{ print $2 }')
```
Comment (OXINARF) [inline Tools/scripts/install-prereqs-ubuntu.sh]: No need to go through awk, just use the the *-s* flag to *lsb_release*

**Comment by lucasdemarchi** [comment]
> arduino-core ? seems not need

can be removed

> curl needed for ?

dunno, can be removed it seems

> droneapi needed for ? -> On 27/06/2016, it makes pip2 crash ...

fine from me

> catkin_pkg needed for ? It is ROS related but actually we don't use ROS.

same

> g++ ccache should be in base_pkgs

ok

**Comment by lucasdemarchi** [comment]
> libtool is present in px4-devguide : http://dev.px4.io/starting-installing-linux.html, but I don't see why it is need else than jsbsim

yep, jsbsim

**Comment by lucasdemarchi** [comment]
> On ubuntu 16.04, default gcc-arm-none-eabi from ubuntu repo is working (4.9.3), but on this ppa, there is a version switch to gcc-arm-none-eabi-5 , any news on that (in good or bad )?

for px4 we actually use the toolchain that comes from arm embedded repository:
https://launchpad.net/gcc-arm-embedded

**Pull request closed without merging**